### DISCUSSION TITLE
https://johanneskonings.dev/notes/2024-06-16-cdk/

### DISCUSSION THREAD

**Discussion by giscus** [JohannesKonings/JohannesKonings.github.io#25] [Note comments]
# https://johanneskonings.dev/notes/2024-06-16-cdk/

https://johanneskonings.dev/notes/2024-06-16-cdk/

**Comment by JohannesKonings** [comment]
Check: https://docs.aws.amazon.com/cdk/v2/guide/policy-validation-synthesis.html
